pkg/kubelet/rkt/container_id.go

```diff
@@ -41,6 +42,14 @@ func buildContainerID(c *containerID) kubecontainer.ContainerID {
 	}
 }
 
+func apiPodToNetPluginContainerID(pod *api.Pod) kubecontainer.ContainerID {
+	return buildContainerID(&containerID{string(pod.UID), pod.Name})
```

Comment:
I'm not seeing how this is causing errors for me here, since AFAICT the fake container ID made here is only used with SetUpPod/TearDownPod/GetNetNS() and it seems to be round-tripped correctly.  I can't see anything that adds "rkt://" to the kubecontainer.ContainerID unless that got added in the last week since I haven't rebased since then.  It could get added if something did string(<ContainerID>) but that's not something that kubenet or rkt.go appear to do in SetUpPod/TearDownPod...  could have missed it though.